packages/vapor/resources/icons/svg/push-32px.svg

```diff
@@ -0,0 +1,5 @@
+<svg viewBox="0 0 32 32" stroke="#282829" fill="none" xmlns="http://www.w3.org/2000/svg">
+<path d="M16 29.5L16 16.5" stroke-linecap="round" stroke-linejoin="round"/>
+<path d="M11 21L16 16L21 21" stroke-linecap="round" stroke-linejoin="round"/>
+<path d="M24.5 23C27.4167 23 29.5 20.2795 29.5 17.5C29.5 14.7204 28.125 12.2984 25.5 12C25.5085 8.06358 22.334 5 18.5 5C15.0312 5 11.5551 7.59435 11 11C8 11 6 13 6 16C4.05005 15.9345 2.5 17.4914 2.5 19.5C2.5 21.4882 4.40034 23 6.33333 23H7.5" stroke="#282829" stroke-linecap="round" stroke-linejoin="round"/>
```

Comment:
If we have `stroke-linecap="round" stroke-linejoin="round"` in both 3 `path`, then they could be moved to the root `svg`.